tests/v3/connection_state_test.c

```diff
@@ -243,14 +244,16 @@ static int s_setup_mqtt_server_fn(struct aws_allocator *allocator, void *ctx) {
     ASSERT_SUCCESS(aws_condition_variable_init(&state_test_data->cvar));
     ASSERT_SUCCESS(aws_mutex_init(&state_test_data->lock));
 
+    /* Sleep for one millisecond to insure timestamp doesn't repeat over tests */
+    aws_thread_current_sleep(ONE_MILLISECOND);
     uint64_t timestamp = 0;
     ASSERT_SUCCESS(aws_sys_clock_get_ticks(&timestamp));
 
     snprintf(
         state_test_data->endpoint.address,
         sizeof(state_test_data->endpoint.address),
         LOCAL_SOCK_TEST_PATTERN,
-        (long long unsigned)timestamp);
+        (long long unsigned)timestamp); // STEVE TODO maybe use a UUID here instead or add a UUID to the timestamp
```

Comment:
Nitpick: Remove TODO?

Reply:
oops!

Reply:
I realize we have LOTS of tests, across lots of C repos with this pattern (use timestamp as socket name)

Your TODO is right, that we'd be better off using UUID, instead of a timestamp.
It's possible to run tests in parallel, where this sleep trick might not work, it would still be possible for multiple tests to be hitting this same line of code within the same millisecond.

Anyway, if this fixes your flaky test for now ... whatever. But if we keep encountering this kind of thing in the future, the UUID approach would be better